Library/Homebrew/cmd/update-report.rb

```diff
@@ -335,6 +313,83 @@ def link_completions_manpages_and_docs(repository = HOMEBREW_REPOSITORY)
   def migrate_gcc_dependents_if_needed
     # do nothing
   end
+
+  def analytics_message
+    if !Utils::Analytics.messages_displayed? &&
+       !Utils::Analytics.disabled? &&
+       !Utils::Analytics.no_message_output?
+
+      ENV["HOMEBREW_NO_ANALYTICS_THIS_RUN"] = "1"
+      # Use the shell's audible bell.
+      print "\a"
+
+      # Use an extra newline and bold to avoid this being missed.
+      ohai "Homebrew has enabled anonymous aggregate formula and cask analytics."
+      puts <<~EOS
+        #{Tty.bold}Read the analytics documentation (and how to opt-out) here:
+          #{Formatter.url("https://docs.brew.sh/Analytics")}#{Tty.reset}
+        No analytics have been recorded yet (nor will be during this `brew` run).
+
+      EOS
+
+      # Consider the messages possibly missed if not a TTY.
+      Utils::Analytics.messages_displayed! if $stdout.tty?
+    elsif Utils::Analytics.disabled?
+      ohai "Homebrew's analytics are moving to a InfluxDB instance in the EU."
```

Comment:
**suggestion:** I think we can sell this better:
```suggestion
      ohai "Homebrew's anonymous analytics are posted to an InfluxDB instance in the EU as of 2023-06-16."
```

Reply:
Another stab:

```suggestion
      ohai "As of 2023-06-16, brew sends anonymous analytics to a Homebrew-owned system located in the EU."
```

Reply:
Thanks @colindean pushed some updates riffing off this.